chapters/tribler/index.md

```diff
@@ -0,0 +1,339 @@
+
+![Content View](images/Context_View.png)
+
+The context view shows the relationships, dependencies and interactions between the system and its environment.
+
+###Important entities
+An important entity is someone or something that is of vital importance to ensure that Tribler is functioning and meeting its responsibilities.
+
+The main entities in Tribler's functionality are the user and ["Dispersy"](https://github.com/Tribler/dispersy). Dispersy is an elastic database system designed for P2P-like scenarios. The database obtains torrent-files from the user, and the user can select downloadable torrent-files from the database managed by Dispersy.
+
+Dispersy was originally only a database system, but has been adapted to setup connections with peers as well. It discovers potential peers and tries to connect to them. If the peer happens to be behind a NAT protocol, Dispersy will attempt to puncture this and connect afterwards.
+
+Being specifically for Tribler, it is hard to classify Dispersy. Being a completely separate project, it could be seen as an external entity. However, its niche 'user' base could classify it as an internal entity as well.
+
+An important external entity ensuring Tribler's current functionality is the BitTorrent Streaming Protocol, which allows users to watch content without downloading, providing the Youtube-like service.
+
+Being heavily inspired by Tor, Tribler closely follows progressions in the Tor project. If a vulnerability is discovered in the Tor Networking Protocol, it could mean that Tribler's anonymity protocol is compromised as well. While the developers of Tribler have made their own implementation of Tor's protocol, Tor is still of great importance to the project, and could therefore be classified as an important external entity.
+
+While Tribler is already functional, it is still actively being developed, therefore the context view also contains many development-related entities. 
+
+For development Tribler depends on 4 external entities: Python, Freenode, Jenkins and Github. Communication between developers, when not done in person, is done through FreeNode. The code base is being developed in Python, utilizing some of Python's libraries, and is managed on [GitHub](https://github.com/Tribler/tribler). Before pull-requests containing additions/adaptions to the code base, get accepted, these changes have to be tested, this is where Jenkins comes in.
+
+Other than the external entities that Tribler depends heavily on for its development, main bodies of interest are the developers and the project leader, as they are putting it all together.
+
+Tribler is a multi-platform program, build to run on Windows, Mac OS and Linux systems. Development on the other hand is only supported on Linux due to issues with libraries that are needed for development. 
+
+##Security perspective
```

Comment:
This section could benefit a bit from explaining the issues with anonymity in Tribler as well. One could otherwise argue

> "If for users of Tribler anonymity is so important, why rely on software which has some flaws in its Tor-like implementation?"